lib/services/subscription.js

```diff
@@ -1,31 +1,34 @@
-const errorHandler = require('../error-handler');
-const normalizeQuery = require('../normalize-query');
-const Base = require('./base');
+const errorHandler = require("../error-handler");
+const normalizeQuery = require("../normalize-query");
+const Base = require("./base");
```

Comment:
Formatting should say consistent with the linter set up in the repository (it should fix automatically if you run `npm test` locally and the commit).

Reply:
Sorry about that, should be ok now.